src/components/MiniMap/MiniMap.tsx

```diff
@@ -28,71 +19,81 @@ interface MiniMapProperties {
   setMapOption: (input: string) => {};
 }
 
+const getCounty = (stateId: string, fullFips: string) =>
+  _find(
+    // @ts-ignore: US_STATE_DATASET is .js, but this is valid
+    US_STATE_DATASET.state_county_map_dataset[stateId].county_dataset,
+    ['full_fips_code', fullFips],
+  );
```

Comment:
We are doing this on [LocationPage.js#L30](https://github.com/covid-projections/covid-projections/blob/4b0d75de7a5ccb53e2709464bbb1514a95908474/src/screens/LocationPage/LocationPage.js#L30) too, should we update `setSelectedCounty` so it takes the FIPS instead of the county object?

Reply:
Yeah it looks like its only used to get the fips so that would work (or could add a TODO and get this merged in for the time being)